### PR TITLE
fix: don't shorten ens names

### PR DESCRIPTION
**Pull request by jjramirezn**
Check everywhere when the addresses are shortened if we have an ens. If we have a ens we should not shorten it

### CODE REVIEW

File: src/components/Create/Link/Success.view.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export const CreateLinkSuccessView = ({
             {link && <QRCodeWrapper url={link} />}
             <label className="text-h8 ">
                 {createType === 'direct'
-                    ? `You have successfully sent the funds to ${recipient.name?.endsWith('.eth') ? recipient.name : utils.shortenAddressLong(recipient.address ?? '')}.`
+                    ? `You have successfully sent the funds to ${recipient.name?.endsWith('.eth') ? recipient.name : utils.printableAddress(recipient.address ?? '')}.`
```
[the remark this answers is not in the document]
Comment (jjramirezn) [reply src/components/Create/Link/Success.view.tsx:89]: The remaining uses are for transaction hashes so it doesn't apply

[the remark this answers is not in the document]
Comment (jjramirezn) [reply src/components/Create/Link/Success.view.tsx:89]: The remaining uses are for transaction hashes so it doesn't apply

File: src/components/Create/Link/Input.view.tsx [hunk 2]
```diff
@@ -241,7 +241,7 @@ export const CreateLinkInputView = ({
                 {createType === 'link'
                     ? 'Text Tokens'
                     : createType === 'direct'
-                      ? `Send to ${recipient.name?.endsWith('.eth') ? recipient.name : utils.shortenAddressLong(recipient.address ?? '')}`
+                      ? `Send to ${recipient.name?.endsWith('.eth') ? recipient.name : utils.printableAddress(recipient.address ?? '')}`
                       : `Send to ${recipient.name}`}
             </label>
```
[the remark this answers is not in the document]
Comment (jjramirezn) [reply src/components/Create/Link/Input.view.tsx:246]: does not apply to all instances because sometimes is used for transaction hashes

**Comment by dushebaa** [comment]
`printableAddress(undefined)` throws an error due to removed null check

**Review by Hugo0 — APPROVED** [review]
nice and simple

Thought: Ideally we should standardize how we treat addresses across the app

ie, if they have a penaut name or ens name, show that, if not, shortened address, and have it be copiable

Stuff for down the line

**Pull request merged**